layers/+lang/lean/packages.el

```diff
@@ -0,0 +1,42 @@
+;;; packages.el --- Lean Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2014-2015 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+(setq lean-packages '(mmm-mode 
+                      f 
+                      dash-functional
```

Comment:
Okay, I was incorrect. I just looked at recipe of both packages and indeed only required files are pulled in according to packages from same github repo. So you'll need to specifically add `dash-functional` if it's needed.

I do have `dash-functional` but it's installed as dependency of `company-tern`.